php-classes/Emergence/People/InvitationsRequestHandler.php

```diff
@@ -81,41 +88,44 @@ public static function handleSendRequest()
     {
         $GLOBALS['Session']->requireAccountLevel('Staff');
 
-        if (empty($_POST['people']) || !is_array($_POST['people'])) {
+        $requestData = \JSON::getRequestData();
```

Comment:
This would break any existing usage of the invitations API

Do `$requestData = JSON::getRequestData() ?: $_POST`